Makefile

```diff
@@ -237,11 +237,18 @@ binary-bundle: xp-fleet xp-fleetctl
 
 # Build orbit/fleetd fleetd_tables extension
 fleetd-tables-windows:
-	GOOS=windows GOARCH=amd64 go build -o fleetd_tables_windows.ext ./orbit/cmd/fleetd_tables
+	GOOS=windows GOARCH=amd64 go build -o fleetd_tables_windows.exe ./orbit/cmd/fleetd_tables
```

Comment:
Nit: Curious to know the reason to change the extension for Windows.

Reply:
Windows should be .exe not .ext. see macadmin and Kolide's extensions